css/functions.json

```diff
@@ -551,15 +551,15 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/gradient/repeating-radial-gradient"
   },
   "rgb()": {
-    "syntax": "rgb( <percentage>{3} [ / <alpha-value> ]? ) | rgb( <number>{3} [ / <alpha-value> ]? ) | rgb( <percentage>#{3} , <alpha-value>? ) | rgb( <number>#{3} , <alpha-value>? )",
+    "syntax": "rgb( [ <number> | <percentage> | none ]{3} [ / [ <alpha-value> | none ] ]?  ) | rgb( <percentage>#{3} , <alpha-value>? ) | rgb( <number>#{3} , <alpha-value>? )",
```

Comment:
Just noting that this puts:

```
rgb() = [ <modern-rgb-syntax> | <legacy-rgb-syntax> ]
```

instead of this order that the spec has:

```
rgb() = [ <legacy-rgb-syntax> | <modern-rgb-syntax>]
```